### PR TITLE
add pytest for configurationfiles utils

### PR DESCRIPTION
**Pull request by awirb**
* add check if folder exists, if input parameters are in correct format
*  add check if configuration files are found

### CODE REVIEW

File: avaframe/tests/test_cfgUtils.py [hunk 1]
```diff
@@ -52,3 +55,83 @@ def test_cfgHash():
...
+    simNames = cfgUtils.filterSims(avaDir, parametersDict, specDir='')
+
+    testRel = False
+    testRel2 = False
+    if 'release1HS' in simNames[0]:
+        testRel = True
+    if 'release2HS' in simNames[0]:
+        testRel2 = True
+
+    assert testRel == True
```
Comment (matthiasto) [inline avaframe/tests/test_cfgUtils.py:135]: you dont need the == True for a boolean, assert testRel should be enough

**Review by matthiasto — APPROVED** [review]

File: avaframe/tests/test_cfgUtils.py [hunk 2]
```diff
@@ -52,3 +55,83 @@ def test_cfgHash():
...
+    assert cfgDict['TEST1']['mu'] == cfg['TEST1'].getfloat('mu')
+
+
+def test_orderSimFiles():
+    """ test generating order of simulation results """
+
+    avaTestDir = 'avaHockeyChannelPytest'
+    dirPath = os.path.dirname(__file__)
+    avaDir = os.path.join(dirPath, '..', '..', 'benchmarks', avaTestDir)
+    inputDir = os.path.join(avaDir, 'Outputs', 'com1DFA', 'peakFiles')
```
Comment (fso42) [inline avaframe/tests/test_cfgUtils.py:93]: I suggest switching all the os.path.joins to pathlib functions (also further down)

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/tests/test_cfgUtils.py (hunk 2 above)
Comment (awirb) [reply avaframe/tests/test_cfgUtils.py]: now all paths in here using pathlib

File: avaframe/tests/test_cfgUtils.py (hunk 1 above)
Comment (awirb) [reply avaframe/tests/test_cfgUtils.py]: done

**Review by fso42 — APPROVED** [review]

**Pull request merged**